---
geometry_msgs/PoseArray grasp_points
